sdk/nodejs/policy/server.ts

```diff
@@ -117,7 +121,9 @@ export function serve(
     servingPolicyPack = policyPackName;
 
     // Finally connect up the gRPC client/server and listen for incoming requests.
-    const server = new grpc.Server();
+    const server = new grpc.Server(<ChannelOptions>{
+        "grpc.max_receive_message_length": maxMessageSize,
+    });
```

Comment:
Is the `<ChannelOptions>` cast necessary? Why not simply:

```typescript
    const server = new grpc.Server({
        "grpc.max_receive_message_length": maxMessageSize,
    });
```

?

Reply:
`grpc.Server()` expects the `ChannelOptions` type so I either need need to cast it in the invoke or declare it like:
```
const grpcChannelOptions: ChannelOptions = {
    "grpc.max_receive_message_length": maxMessageSize,
};
const server = new grpc.Server(grpcChannelOptions);
```

Reply:
The TypeScript compiler actually complains if you don't specify the type?

Reply:
I tried copying a minimal version of these types in a TypeScript [playground](https://www.typescriptlang.org/v2/en/play?#code/PTAEAsBdIBwZwFwgOYEtLgK4CMB0BjAewFthkAnGfMy-AWgDtCATAU2GwBtDtgB2AJzMAzPgBmrACysAbAEMAHMPnC5k7AEYADOK1apYgQFYjGhUYBMwzTK3AYc-AGs5yVnBpU6AKw9xy1PjgcgwMrJx0hDCQqIQMcLiQcADEADIWGnSpygBQrAAeMITkkKCoDJCs5GKOrKAAwsGh4QDy0bHxoADeOaB9oADkFFS4cHCcAPqQcuRukBMMcsSsE4QAblXkqGwDAPwIoHCQWwzIANy9-UO0uDBbxDMAnhOYcFUTrqwVewdHJ+eXPrXEZvIgMZhPF5vcgfNzffaHY7lAH9QbDfC4Ng1TCceZyTAYYroR4-RH-C6o4EYpysVgOTioDZTVDLCbEOCkhiYYjYKoUq7o3A0ulyBlMmLLQgEtkchFcnl8wFom7QtaofArMFiVDIUl-ZH8oGCh75CZg-CYcjkL7zP6sJayg7y3nkQ3KkbldCoUUTa1gsL4ebYRxOQhiMQyzncl1uqm4E2+1j+pNBkNhiPsqMK11KuOvFbcfA+uA4IIhMKTIqEThZmO541yU1vcFs9xwT4TThfZAYWuKykN01+1iMlbLMYdrunXty6P9-oAbRpj1+SNOAF1V-9QAAfUDOqq70CYcGsbVhZgXAC+ORyIAg0HgSDI6CweCIpHRnnoTDYHG4vCCCI4hSLIijKHIqjqNouj6JIhgmGYljWBotj2CGnweOiPh+AEwCqlUiQpKkGgaHweSFMUpT4JwchjKAADKVQbOQ3RKncjJyJUoBRDEcSIA0TQVm0fHxBcSpgn8mCBsUAAUvEdHACKNOWrTtPxACUbGoqiGCoAkCn8aAAC8PHqZ0uy7N0V5ujeN45JJpQmgAsm2nwMagABedSmdoFiSKAABUoB+QFwWSHo4mOYczGHqZYQAO6MbF5CyT0qIAESDomGqjq2E5uJ23YYBlBwuW5bged5AA0ORXhpFxAA), and the compiler lets me create the server instance without specifying the `ChannelOptions` type.

```typescript
// https://github.com/grpc/grpc-node/blob/79d3cfe4e6a836a3a4b10cf00e4f95518523b160/packages/grpc-js/src/channel-options.ts#L21-L36
export interface ChannelOptions {
    'grpc.ssl_target_name_override'?: string;
    'grpc.primary_user_agent'?: string;
    'grpc.secondary_user_agent'?: string;
    'grpc.default_authority'?: string;
    'grpc.keepalive_time_ms'?: number;
    'grpc.keepalive_timeout_ms'?: number;
    'grpc.service_config'?: string;
    'grpc.max_concurrent_streams'?: number;
    'grpc.initial_reconnect_backoff_ms'?: number;
    'grpc.max_reconnect_backoff_ms'?: number;
    'grpc.use_local_subchannel_pool'?: number;
    'grpc.max_send_message_length'?: number;
    'grpc.max_receive_message_length'?: number;
    [key: string]: string | number | undefined;
}

// https://github.com/grpc/grpc-node/blob/79d3cfe4e6a836a3a4b10cf00e4f95518523b160/packages/grpc-js/src/server.ts#L117
export class Server {
    private options: ChannelOptions;

    constructor(options?: ChannelOptions) {
        this.options = options ?? {};
    }
}

const maxMessageSize = 1024 * 1024 * 400;

const server = new Server({
    "grpc.max_receive_message_length": maxMessageSize,
});
```

Reply:
yeah you're right. I could've sworn it was giving me problems earlier 🤷‍♂️